packages/rspack-test-tools/tests/configCases/library/modern-module-named-import-externals/rspack.config.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+	mode: "none",
+	entry: { main: "./index.js", test: "./test.js" },
+	output: {
+		module: true,
+		library: {
+			type: "modern-module"
+		},
+		filename: "[name].js",
+		chunkFormat: "module"
+	},
+	experiments: {
+		outputModule: true
+	},
+	resolve: {
+		extensions: [".js"]
+	},
+	externalsType: "module",
+	externals: [
+		"externals0",
+		"externals1",
+		"externals2",
+		"externals3",
+		"externals4"
+	],
+	optimization: {
+		concatenateModules: true,
+		usedExports: true
+	},
+	plugins: [
+		function () {
+			const handler = compilation => {
+				compilation.hooks.processAssets.tap("testcase", assets => {
+					const source = assets["test.js"].source();
+					expect(source).toMatchInlineSnapshot(`
+				import { a, b } from "externals0";
+				import { a as a_0 } from "externals1";
+				import default_0 from "externals2";
+				import * as __WEBPACK_EXTERNAL_MODULE_externals3__ from "externals3";
```

Comment:
I think we could remove `__WEBPACK_EXTERNAL_MODULE_` prefix, it's way too long.

Reply:
I think It's good, we can make it in another PR, as it will change a lots of snapshot, I want to keep this PR simpler

Reply:
Got it.